lib/Dialect/FIRRTL/InnerSymbolTable.cpp

```diff
@@ -186,17 +223,19 @@ Operation *InnerRefNamespace::lookupOp(hw::InnerRefAttr inner) {
 }
 
 //===----------------------------------------------------------------------===//
-// InnerRef verification
+// InnerRefNamespace verification
 //===----------------------------------------------------------------------===//
 
 namespace detail {
 
-LogicalResult verifyInnerRefs(Operation *op) {
+LogicalResult verifyInnerRefNamespace(Operation *op) {
   // Construct the symbol tables.
   InnerSymbolTableCollection innerSymTables;
   SymbolTable symbolTable(op);
```

Comment:
Should the `SymbolTable` verifier run before the `InnerRefNamespace` verifier?  https://github.com/llvm/llvm-project/blob/main/mlir/include/mlir/IR/OpBase.td#L1732